[2016-09-21 09:05:50,373] ERROR [play-thread-1] com.aton.base.secure.Secure.checkAuthentication(Secure.java:92) - Login admin not exists,admin_id=null
[2016-09-21 09:28:26,172] ERROR [play-thread-1] com.aton.base.secure.Secure.checkAuthentication(Secure.java:124) - Login user not exists,user_id=1689
[2016-09-21 09:52:44,657] ERROR [jobs-thread-6] play.Logger.niceThrowable(Logger.java:604) - 

@71en7c07c
Error during job execution (jobs.TaskKeeper)

Compilation error (In /app/models/Task3.java around line 871)
The file /app/models/Task3.java could not be compiled. Error raised is : taskMapper cannot be resolved

play.exceptions.CompilationException: taskMapper cannot be resolved
	at play.classloading.ApplicationCompiler$2.acceptResult(ApplicationCompiler.java:256)
	at org.eclipse.jdt.internal.compiler.Compiler.handleInternalException(Compiler.java:678)
	at org.eclipse.jdt.internal.compiler.Compiler.compile(Compiler.java:522)
	at play.classloading.ApplicationCompiler.compile(ApplicationCompiler.java:292)
	at play.classloading.ApplicationClasses$ApplicationClass.compile(ApplicationClasses.java:289)
	at play.classloading.ApplicationClassloader.detectChanges(ApplicationClassloader.java:326)
	at play.Play.detectChanges(Play.java:636)
	at play.Invoker$Invocation.init(Invoker.java:199)
	at play.jobs.Job.call(Job.java:185)
	at Invocation.Job(Play!)
[2016-09-21 09:52:44,811] ERROR [jobs-thread-1] play.Logger.niceThrowable(Logger.java:604) - 

@71en7c07e
Error during job execution (jobs.TaskKeeper2)

Compilation error (In /app/models/Task3.java around line 871)
The file /app/models/Task3.java could not be compiled. Error raised is : taskMapper cannot be resolved

play.exceptions.CompilationException: taskMapper cannot be resolved
	at play.classloading.ApplicationCompiler$2.acceptResult(ApplicationCompiler.java:256)
	at org.eclipse.jdt.internal.compiler.Compiler.handleInternalException(Compiler.java:678)
	at org.eclipse.jdt.internal.compiler.Compiler.compile(Compiler.java:522)
	at play.classloading.ApplicationCompiler.compile(ApplicationCompiler.java:292)
	at play.classloading.ApplicationClasses$ApplicationClass.compile(ApplicationClasses.java:289)
	at play.classloading.ApplicationClassloader.detectChanges(ApplicationClassloader.java:326)
	at play.Play.detectChanges(Play.java:636)
	at play.Invoker$Invocation.init(Invoker.java:199)
	at play.jobs.Job.call(Job.java:185)
	at Invocation.Job(Play!)
[2016-09-21 09:58:20,076] ERROR [play-thread-1] com.aton.base.secure.Secure.checkAuthentication(Secure.java:92) - Login admin not exists,admin_id=null
[2016-09-21 10:00:06,236] ERROR [play-thread-1] com.aton.base.BaseController.handleServerError(BaseController.java:232) - =======>Server Fatal Error<======
[2016-09-21 10:00:06,237] ERROR [play-thread-1] com.aton.base.BaseController.handleServerError(BaseController.java:233) - 
java.lang.NullPointerException
	at controllers.UserCenter.shopDetail2(UserCenter.java:372)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:39)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:25)
	at java.lang.reflect.Method.invoke(Method.java:597)
	at play.mvc.ActionInvoker.invokeWithContinuation(ActionInvoker.java:524)
	at play.mvc.ActionInvoker.invoke(ActionInvoker.java:475)
	at play.mvc.ActionInvoker.invokeControllerMethod(ActionInvoker.java:451)
	at play.mvc.ActionInvoker.invokeControllerMethod(ActionInvoker.java:446)
	at play.mvc.ActionInvoker.invoke(ActionInvoker.java:160)
	at play.server.PlayHandler$NettyInvocation.execute(PlayHandler.java:267)
	at play.Invoker$Invocation$1.apply(Invoker.java:290)
	at play.Invoker$Invocation$1.apply(Invoker.java:287)
	at play.db.jpa.JPA.withTransaction(JPA.java:258)
	at play.db.jpa.JPA.withinFilter(JPA.java:217)
	at play.db.jpa.JPAPlugin$TransactionalFilter.withinFilter(JPAPlugin.java:299)
	at play.Invoker$Invocation.withinFilter(Invoker.java:271)
	at play.Invoker$Invocation.run(Invoker.java:287)
	at play.server.PlayHandler$NettyInvocation.run(PlayHandler.java:245)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:439)
	at java.util.concurrent.FutureTask$Sync.innerRun(FutureTask.java:303)
	at java.util.concurrent.FutureTask.run(FutureTask.java:138)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$301(ScheduledThreadPoolExecutor.java:98)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:206)
	at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:895)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:918)
	at java.lang.Thread.run(Thread.java:662)
[2016-09-21 10:13:38,593] ERROR [play-thread-1] play.Logger.niceThrowable(Logger.java:604) - 

@71en7c07g
Internal Server Error (500) for request GET /seller/task3/778415068386689024

Template execution error (In /app/views/TaskCenter/taskDetail3.html around line 725)
Execution error occured in template /app/views/TaskCenter/taskDetail3.html. Exception raised was MissingPropertyException : No such property: genderIngot for class: models.Task3.

play.exceptions.TemplateExecutionException: No such property: genderIngot for class: models.Task3
	at play.templates.BaseTemplate.throwException(BaseTemplate.java:86)
	at play.templates.GroovyTemplate.internalRender(GroovyTemplate.java:285)
	at play.templates.Template.render(Template.java:26)
	at play.templates.GroovyTemplate.render(GroovyTemplate.java:208)
	at play.mvc.results.RenderTemplate.<init>(RenderTemplate.java:24)
	at play.mvc.Controller.renderTemplate(Controller.java:676)
	at play.mvc.Controller.renderTemplate(Controller.java:656)
	at play.mvc.Controller.render(Controller.java:713)
	at controllers.TaskCenter.taskDetail3(TaskCenter.java:1786)
	at play.mvc.ActionInvoker.invokeWithContinuation(ActionInvoker.java:524)
	at play.mvc.ActionInvoker.invoke(ActionInvoker.java:475)
	at play.mvc.ActionInvoker.invokeControllerMethod(ActionInvoker.java:451)
	at play.mvc.ActionInvoker.invokeControllerMethod(ActionInvoker.java:446)
	at play.mvc.ActionInvoker.invoke(ActionInvoker.java:160)
	at Invocation.HTTP Request(Play!)
Caused by: groovy.lang.MissingPropertyException: No such property: genderIngot for class: models.Task3
	at play.templates.GroovyTemplate.internalRender(GroovyTemplate.java:260)
	... 13 more
[2016-09-21 10:13:48,388] ERROR [play-thread-1] play.Logger.niceThrowable(Logger.java:604) - 

@71en7c07h
Internal Server Error (500) for request GET /seller/task3/778415068386689024

Template execution error (In /app/views/TaskCenter/taskDetail3.html around line 725)
Execution error occured in template /app/views/TaskCenter/taskDetail3.html. Exception raised was MissingPropertyException : No such property: genderIngot for class: models.Task3.

play.exceptions.TemplateExecutionException: No such property: genderIngot for class: models.Task3
	at play.templates.BaseTemplate.throwException(BaseTemplate.java:86)
	at play.templates.GroovyTemplate.internalRender(GroovyTemplate.java:285)
	at play.templates.Template.render(Template.java:26)
	at play.templates.GroovyTemplate.render(GroovyTemplate.java:208)
	at play.mvc.results.RenderTemplate.<init>(RenderTemplate.java:24)
	at play.mvc.Controller.renderTemplate(Controller.java:676)
	at play.mvc.Controller.renderTemplate(Controller.java:656)
	at play.mvc.Controller.render(Controller.java:713)
	at controllers.TaskCenter.taskDetail3(TaskCenter.java:1786)
	at play.mvc.ActionInvoker.invokeWithContinuation(ActionInvoker.java:524)
	at play.mvc.ActionInvoker.invoke(ActionInvoker.java:475)
	at play.mvc.ActionInvoker.invokeControllerMethod(ActionInvoker.java:451)
	at play.mvc.ActionInvoker.invokeControllerMethod(ActionInvoker.java:446)
	at play.mvc.ActionInvoker.invoke(ActionInvoker.java:160)
	at Invocation.HTTP Request(Play!)
Caused by: groovy.lang.MissingPropertyException: No such property: genderIngot for class: models.Task3
	at /app/views/TaskCenter/taskDetail3.html.(line:725)
	at play.templates.GroovyTemplate.internalRender(GroovyTemplate.java:260)
	... 13 more
[2016-09-21 10:13:57,731] ERROR [play-thread-1] play.Logger.niceThrowable(Logger.java:604) - 

@71en7c07i
Internal Server Error (500) for request GET /seller/task3/778415068386689024

Template execution error (In /app/views/TaskCenter/taskDetail3.html around line 725)
Execution error occured in template /app/views/TaskCenter/taskDetail3.html. Exception raised was MissingPropertyException : No such property: genderIngot for class: models.Task3.

play.exceptions.TemplateExecutionException: No such property: genderIngot for class: models.Task3
	at play.templates.BaseTemplate.throwException(BaseTemplate.java:86)
	at play.templates.GroovyTemplate.internalRender(GroovyTemplate.java:285)
	at play.templates.Template.render(Template.java:26)
	at play.templates.GroovyTemplate.render(GroovyTemplate.java:208)
	at play.mvc.results.RenderTemplate.<init>(RenderTemplate.java:24)
	at play.mvc.Controller.renderTemplate(Controller.java:676)
	at play.mvc.Controller.renderTemplate(Controller.java:656)
	at play.mvc.Controller.render(Controller.java:713)
	at controllers.TaskCenter.taskDetail3(TaskCenter.java:1786)
	at play.mvc.ActionInvoker.invokeWithContinuation(ActionInvoker.java:524)
	at play.mvc.ActionInvoker.invoke(ActionInvoker.java:475)
	at play.mvc.ActionInvoker.invokeControllerMethod(ActionInvoker.java:451)
	at play.mvc.ActionInvoker.invokeControllerMethod(ActionInvoker.java:446)
	at play.mvc.ActionInvoker.invoke(ActionInvoker.java:160)
	at Invocation.HTTP Request(Play!)
Caused by: groovy.lang.MissingPropertyException: No such property: genderIngot for class: models.Task3
	at /app/views/TaskCenter/taskDetail3.html.(line:725)
	at play.templates.GroovyTemplate.internalRender(GroovyTemplate.java:260)
	... 13 more
[2016-09-21 10:15:17,960] ERROR [play-thread-1] play.Logger.niceThrowable(Logger.java:604) - 

@71en7c07j
Internal Server Error (500) for request GET /seller/task3/778415068386689024

Template execution error (In /app/views/TaskCenter/taskDetail3.html around line 729)
Execution error occured in template /app/views/TaskCenter/taskDetail3.html. Exception raised was MissingPropertyException : No such property: buyerLocationIngot for class: models.Task3.

play.exceptions.TemplateExecutionException: No such property: buyerLocationIngot for class: models.Task3
	at play.templates.BaseTemplate.throwException(BaseTemplate.java:86)
	at play.templates.GroovyTemplate.internalRender(GroovyTemplate.java:285)
	at play.templates.Template.render(Template.java:26)
	at play.templates.GroovyTemplate.render(GroovyTemplate.java:208)
	at play.mvc.results.RenderTemplate.<init>(RenderTemplate.java:24)
	at play.mvc.Controller.renderTemplate(Controller.java:676)
	at play.mvc.Controller.renderTemplate(Controller.java:656)
	at play.mvc.Controller.render(Controller.java:713)
	at controllers.TaskCenter.taskDetail3(TaskCenter.java:1786)
	at play.mvc.ActionInvoker.invokeWithContinuation(ActionInvoker.java:524)
	at play.mvc.ActionInvoker.invoke(ActionInvoker.java:475)
	at play.mvc.ActionInvoker.invokeControllerMethod(ActionInvoker.java:451)
	at play.mvc.ActionInvoker.invokeControllerMethod(ActionInvoker.java:446)
	at play.mvc.ActionInvoker.invoke(ActionInvoker.java:160)
	at Invocation.HTTP Request(Play!)
Caused by: groovy.lang.MissingPropertyException: No such property: buyerLocationIngot for class: models.Task3
	at /app/views/TaskCenter/taskDetail3.html.(line:729)
	at play.templates.GroovyTemplate.internalRender(GroovyTemplate.java:260)
	... 13 more
[2016-09-21 10:16:16,464] ERROR [jobs-thread-5] play.Logger.niceThrowable(Logger.java:604) - 

@71en7c07k
Error during job execution (jobs.TaskKeeper)

Oops: OutOfMemoryError
An unexpected error occured caused by exception OutOfMemoryError: PermGen space

play.exceptions.UnexpectedException: Unexpected Error
	at play.Invoker$Invocation.onException(Invoker.java:245)
	at play.jobs.Job.onException(Job.java:154)
	at com.aton.job.BaseJob.onException(BaseJob.java:49)
	at play.jobs.Job.call(Job.java:224)
	at Invocation.Job(Play!)
Caused by: java.lang.OutOfMemoryError: PermGen space
	at java.lang.ClassLoader.defineClass1(Native Method)
	at java.lang.ClassLoader.defineClassCond(ClassLoader.java:631)
	at java.lang.ClassLoader.defineClass(ClassLoader.java:615)
	at play.classloading.ApplicationClassloader.loadApplicationClass(ApplicationClassloader.java:152)
	at play.classloading.ApplicationClassloader.getAllClasses(ApplicationClassloader.java:433)
	at play.Play.start(Play.java:527)
	at play.Play.detectChanges(Play.java:653)
	at play.Invoker$Invocation.init(Invoker.java:199)
	at play.jobs.Job.call(Job.java:185)
	... 1 more
[2016-09-21 10:16:18,611] ERROR [jobs-thread-2] play.Logger.niceThrowable(Logger.java:604) - 

@71en7c07m
Error during job execution (jobs.TaskKeeper2)

Oops: OutOfMemoryError
An unexpected error occured caused by exception OutOfMemoryError: PermGen space

play.exceptions.UnexpectedException: Unexpected Error
	at play.Invoker$Invocation.onException(Invoker.java:245)
	at play.jobs.Job.onException(Job.java:154)
	at com.aton.job.BaseJob.onException(BaseJob.java:49)
	at play.jobs.Job.call(Job.java:224)
	at Invocation.Job(Play!)
Caused by: java.lang.OutOfMemoryError: PermGen space
	at java.lang.ClassLoader.defineClass1(Native Method)
	at java.lang.ClassLoader.defineClassCond(ClassLoader.java:631)
	at java.lang.ClassLoader.defineClass(ClassLoader.java:615)
	at play.classloading.ApplicationClassloader.loadApplicationClass(ApplicationClassloader.java:152)
	at play.classloading.ApplicationClassloader.getAllClasses(ApplicationClassloader.java:433)
	at play.Play.start(Play.java:527)
	at play.Play.detectChanges(Play.java:653)
	at play.Invoker$Invocation.init(Invoker.java:199)
	at play.jobs.Job.call(Job.java:185)
	... 1 more
[2016-09-21 10:16:42,333] ERROR [play-thread-1] play.Logger.niceThrowable(Logger.java:604) - 

@71en7c07o
Internal Server Error (500) for request GET /seller/tasks3

Oops: OutOfMemoryError
An unexpected error occured caused by exception OutOfMemoryError: PermGen space

play.exceptions.UnexpectedException: Unexpected Error
	at play.Invoker$Invocation.onException(Invoker.java:245)
	at play.Invoker$Invocation.run(Invoker.java:305)
	at Invocation.HTTP Request(Play!)
Caused by: java.lang.OutOfMemoryError: PermGen space
	at java.lang.ClassLoader.defineClass1(Native Method)
	at java.lang.ClassLoader.defineClassCond(ClassLoader.java:631)
	at java.lang.ClassLoader.defineClass(ClassLoader.java:615)
	at play.classloading.ApplicationClassloader.loadApplicationClass(ApplicationClassloader.java:152)
	at play.classloading.ApplicationClassloader.getAllClasses(ApplicationClassloader.java:433)
	at play.Play.start(Play.java:527)
	at play.Play.detectChanges(Play.java:653)
	at play.Invoker$Invocation.init(Invoker.java:199)
	... 1 more
[2016-09-21 10:16:42,761] ERROR [play-thread-1] play.Logger.error(Logger.java:457) - Error during the 500 response generation
java.lang.OutOfMemoryError: PermGen space
	at java.lang.ClassLoader.defineClass1(Native Method)
	at java.lang.ClassLoader.defineClassCond(ClassLoader.java:631)
	at java.lang.ClassLoader.defineClass(ClassLoader.java:615)
	at play.templates.GroovyTemplate$TClassLoader.defineTemplate(GroovyTemplate.java:79)
	at play.templates.GroovyTemplate.directLoad(GroovyTemplate.java:92)
	at play.templates.BaseTemplate.loadFromCache(BaseTemplate.java:56)
	at play.templates.TemplateLoader.load(TemplateLoader.java:81)
	at play.templates.TemplateLoader.load(TemplateLoader.java:179)
	at play.server.PlayHandler.serve500(PlayHandler.java:792)
	at play.server.PlayHandler$NettyInvocation.run(PlayHandler.java:247)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:439)
	at java.util.concurrent.FutureTask$Sync.innerRun(FutureTask.java:303)
	at java.util.concurrent.FutureTask.run(FutureTask.java:138)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$301(ScheduledThreadPoolExecutor.java:98)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:206)
	at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:895)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:918)
	at java.lang.Thread.run(Thread.java:662)
[2016-09-21 10:17:01,135] ERROR [main] play.Logger.error(Logger.java:426) - Could not bind on port 80
[2016-09-21 10:17:09,249] ERROR [play-thread-1] play.Logger.niceThrowable(Logger.java:604) - 

@71en7c07p
Internal Server Error (500) for request GET /seller/tasks3

Oops: OutOfMemoryError
An unexpected error occured caused by exception OutOfMemoryError: PermGen space

play.exceptions.UnexpectedException: Unexpected Error
	at play.Invoker$Invocation.onException(Invoker.java:245)
	at play.Invoker$Invocation.run(Invoker.java:305)
	at Invocation.HTTP Request(Play!)
Caused by: java.lang.OutOfMemoryError: PermGen space
[2016-09-21 10:17:09,729] ERROR [play-thread-1] play.Logger.error(Logger.java:457) - Error during the 500 response generation
java.lang.OutOfMemoryError: PermGen space
[2016-09-21 10:17:46,325] ERROR [play-thread-1] play.Logger.niceThrowable(Logger.java:604) - 

@71en7c080
Internal Server Error (500) for request GET /seller/tasks3

Oops: OutOfMemoryError
An unexpected error occured caused by exception OutOfMemoryError: PermGen space

play.exceptions.UnexpectedException: Unexpected Error
	at play.Invoker$Invocation.onException(Invoker.java:245)
	at play.Invoker$Invocation.run(Invoker.java:305)
	at Invocation.HTTP Request(Play!)
Caused by: java.lang.OutOfMemoryError: PermGen space
[2016-09-21 10:17:46,766] ERROR [play-thread-1] play.Logger.error(Logger.java:457) - Error during the 500 response generation
java.lang.OutOfMemoryError: PermGen space
[2016-09-21 15:53:41,224] ERROR [jobs-thread-8] play.Logger.niceThrowable(Logger.java:604) - 

@71ep2m4g9
Error during job execution (jobs.TaskKeeper)

Compilation error (In /app/com/aton/config/BizConstants.java around line 53)
The file /app/com/aton/config/BizConstants.java could not be compiled. Error raised is : Type mismatch: cannot convert from double to int

play.exceptions.CompilationException: Type mismatch: cannot convert from double to int
	at play.classloading.ApplicationCompiler$2.acceptResult(ApplicationCompiler.java:256)
	at org.eclipse.jdt.internal.compiler.Compiler.handleInternalException(Compiler.java:678)
	at org.eclipse.jdt.internal.compiler.Compiler.compile(Compiler.java:522)
	at play.classloading.ApplicationCompiler.compile(ApplicationCompiler.java:292)
	at play.classloading.ApplicationClasses$ApplicationClass.compile(ApplicationClasses.java:289)
	at play.classloading.ApplicationClassloader.detectChanges(ApplicationClassloader.java:326)
	at play.Play.detectChanges(Play.java:636)
	at play.Invoker$Invocation.init(Invoker.java:199)
	at play.jobs.Job.call(Job.java:185)
	at Invocation.Job(Play!)
[2016-09-21 15:53:41,335] ERROR [jobs-thread-1] play.Logger.niceThrowable(Logger.java:604) - 

@71ep2m4gb
Error during job execution (jobs.TaskKeeper2)

Compilation error (In /app/com/aton/config/BizConstants.java around line 53)
The file /app/com/aton/config/BizConstants.java could not be compiled. Error raised is : Type mismatch: cannot convert from double to int

play.exceptions.CompilationException: Type mismatch: cannot convert from double to int
	at play.classloading.ApplicationCompiler$2.acceptResult(ApplicationCompiler.java:256)
	at org.eclipse.jdt.internal.compiler.Compiler.handleInternalException(Compiler.java:678)
	at org.eclipse.jdt.internal.compiler.Compiler.compile(Compiler.java:522)
	at play.classloading.ApplicationCompiler.compile(ApplicationCompiler.java:292)
	at play.classloading.ApplicationClasses$ApplicationClass.compile(ApplicationClasses.java:289)
	at play.classloading.ApplicationClassloader.detectChanges(ApplicationClassloader.java:326)
	at play.Play.detectChanges(Play.java:636)
	at play.Invoker$Invocation.init(Invoker.java:199)
	at play.jobs.Job.call(Job.java:185)
	at Invocation.Job(Play!)
[2016-09-21 15:55:41,573] ERROR [play-thread-1] play.Logger.niceThrowable(Logger.java:604) - 

@71ep2m4gd
Internal Server Error (500) for request GET /seller/tasks3

Oops: OutOfMemoryError
An unexpected error occured caused by exception OutOfMemoryError: PermGen space

play.exceptions.UnexpectedException: Unexpected Error
	at play.mvc.ActionInvoker.invoke(ActionInvoker.java:255)
	at Invocation.HTTP Request(Play!)
Caused by: java.lang.OutOfMemoryError: PermGen space
	at java.lang.Class.getDeclaredFields0(Native Method)
	at java.lang.Class.privateGetDeclaredFields(Class.java:2300)
	at java.lang.Class.getDeclaredField(Class.java:1882)
	at play.utils.Java.parameterNames(Java.java:240)
	at play.mvc.ActionInvoker.getActionMethodArgs(ActionInvoker.java:589)
	at play.mvc.ActionInvoker.invokeControllerMethod(ActionInvoker.java:451)
	at play.mvc.ActionInvoker.invokeControllerMethod(ActionInvoker.java:446)
	at play.mvc.ActionInvoker.invoke(ActionInvoker.java:160)
	... 1 more
[2016-09-21 15:55:42,495] ERROR [play-thread-1]  - Oops. Error in Logger !
java.lang.OutOfMemoryError: PermGen space
[2016-09-21 16:01:08,414] ERROR [jobs-thread-2] play.Logger.niceThrowable(Logger.java:604) - 

@71ep3lg3f
Error during job execution (jobs.TaskKeeper)

Compilation error (In /app/models/Task3.java around line 642)
The file /app/models/Task3.java could not be compiled. Error raised is : Syntax error, insert ":: IdentifierOrNew" to complete ReferenceExpression

play.exceptions.CompilationException: Syntax error, insert ":: IdentifierOrNew" to complete ReferenceExpression
	at play.classloading.ApplicationCompiler$2.acceptResult(ApplicationCompiler.java:256)
	at org.eclipse.jdt.internal.compiler.Compiler.compile(Compiler.java:480)
	at play.classloading.ApplicationCompiler.compile(ApplicationCompiler.java:292)
	at play.classloading.ApplicationClasses$ApplicationClass.compile(ApplicationClasses.java:289)
	at play.classloading.ApplicationClassloader.detectChanges(ApplicationClassloader.java:326)
	at play.Play.detectChanges(Play.java:636)
	at play.Invoker$Invocation.init(Invoker.java:199)
	at play.jobs.Job.call(Job.java:185)
	at Invocation.Job(Play!)
[2016-09-21 16:01:08,667] ERROR [jobs-thread-8] play.Logger.niceThrowable(Logger.java:604) - 

@71ep3lg3h
Error during job execution (jobs.TaskKeeper2)

Compilation error (In /app/models/Task3.java around line 642)
The file /app/models/Task3.java could not be compiled. Error raised is : Syntax error, insert ":: IdentifierOrNew" to complete ReferenceExpression

play.exceptions.CompilationException: Syntax error, insert ":: IdentifierOrNew" to complete ReferenceExpression
	at play.classloading.ApplicationCompiler$2.acceptResult(ApplicationCompiler.java:256)
	at org.eclipse.jdt.internal.compiler.Compiler.compile(Compiler.java:480)
	at play.classloading.ApplicationCompiler.compile(ApplicationCompiler.java:292)
	at play.classloading.ApplicationClasses$ApplicationClass.compile(ApplicationClasses.java:289)
	at play.classloading.ApplicationClassloader.detectChanges(ApplicationClassloader.java:326)
	at play.Play.detectChanges(Play.java:636)
	at play.Invoker$Invocation.init(Invoker.java:199)
	at play.jobs.Job.call(Job.java:185)
	at Invocation.Job(Play!)
[2016-09-21 16:03:40,522] ERROR [play-thread-1] models.Task3.save(Task3.java:474) - For input string: "0.1"
java.lang.NumberFormatException: For input string: "0.1"
	at java.lang.NumberFormatException.forInputString(NumberFormatException.java:48)
	at java.lang.Long.parseLong(Long.java:419)
	at java.lang.Long.parseLong(Long.java:468)
	at models.Task3.calculateAndSetTaskPledgeAndIngot(Task3.java:642)
	at models.Task3.save(Task3.java:455)
	at controllers.TaskPublish.saveTask3(TaskPublish.java:642)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:39)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:25)
	at java.lang.reflect.Method.invoke(Method.java:597)
	at play.mvc.ActionInvoker.invokeWithContinuation(ActionInvoker.java:524)
	at play.mvc.ActionInvoker.invoke(ActionInvoker.java:475)
	at play.mvc.ActionInvoker.invokeControllerMethod(ActionInvoker.java:451)
	at play.mvc.ActionInvoker.invokeControllerMethod(ActionInvoker.java:446)
	at play.mvc.ActionInvoker.invoke(ActionInvoker.java:160)
	at play.server.PlayHandler$NettyInvocation.execute(PlayHandler.java:267)
	at play.Invoker$Invocation$1.apply(Invoker.java:290)
	at play.Invoker$Invocation$1.apply(Invoker.java:287)
	at play.db.jpa.JPA.withTransaction(JPA.java:258)
	at play.db.jpa.JPA.withinFilter(JPA.java:217)
	at play.db.jpa.JPAPlugin$TransactionalFilter.withinFilter(JPAPlugin.java:299)
	at play.Invoker$Invocation.withinFilter(Invoker.java:271)
	at play.Invoker$Invocation.run(Invoker.java:287)
	at play.server.PlayHandler$NettyInvocation.run(PlayHandler.java:245)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:439)
	at java.util.concurrent.FutureTask$Sync.innerRun(FutureTask.java:303)
	at java.util.concurrent.FutureTask.run(FutureTask.java:138)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$301(ScheduledThreadPoolExecutor.java:98)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:206)
	at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:895)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:918)
	at java.lang.Thread.run(Thread.java:662)
[2016-09-21 16:04:13,191] ERROR [play-thread-1] models.Task3.save(Task3.java:474) - For input string: "0.1"
java.lang.NumberFormatException: For input string: "0.1"
	at java.lang.NumberFormatException.forInputString(NumberFormatException.java:48)
	at java.lang.Long.parseLong(Long.java:419)
	at java.lang.Long.parseLong(Long.java:468)
	at models.Task3.calculateAndSetTaskPledgeAndIngot(Task3.java:642)
	at models.Task3.save(Task3.java:455)
	at controllers.TaskPublish.saveTask3(TaskPublish.java:642)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:39)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:25)
	at java.lang.reflect.Method.invoke(Method.java:597)
	at play.mvc.ActionInvoker.invokeWithContinuation(ActionInvoker.java:524)
	at play.mvc.ActionInvoker.invoke(ActionInvoker.java:475)
	at play.mvc.ActionInvoker.invokeControllerMethod(ActionInvoker.java:451)
	at play.mvc.ActionInvoker.invokeControllerMethod(ActionInvoker.java:446)
	at play.mvc.ActionInvoker.invoke(ActionInvoker.java:160)
	at play.server.PlayHandler$NettyInvocation.execute(PlayHandler.java:267)
	at play.Invoker$Invocation$1.apply(Invoker.java:290)
	at play.Invoker$Invocation$1.apply(Invoker.java:287)
	at play.db.jpa.JPA.withTransaction(JPA.java:258)
	at play.db.jpa.JPA.withinFilter(JPA.java:217)
	at play.db.jpa.JPAPlugin$TransactionalFilter.withinFilter(JPAPlugin.java:299)
	at play.Invoker$Invocation.withinFilter(Invoker.java:271)
	at play.Invoker$Invocation.run(Invoker.java:287)
	at play.server.PlayHandler$NettyInvocation.run(PlayHandler.java:245)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:439)
	at java.util.concurrent.FutureTask$Sync.innerRun(FutureTask.java:303)
	at java.util.concurrent.FutureTask.run(FutureTask.java:138)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$301(ScheduledThreadPoolExecutor.java:98)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:206)
	at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:895)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:918)
	at java.lang.Thread.run(Thread.java:662)
[2016-09-21 16:16:51,056] ERROR [jobs-thread-2] play.Logger.niceThrowable(Logger.java:604) - 

@71ep5n6fa
Error during job execution (jobs.TaskKeeper)

Compilation error (In /app/models/Task3.java around line 647)
The file /app/models/Task3.java could not be compiled. Error raised is : The operator += is undefined for the argument type(s) Long, double

play.exceptions.CompilationException: The operator += is undefined for the argument type(s) Long, double
	at play.classloading.ApplicationCompiler$2.acceptResult(ApplicationCompiler.java:256)
	at org.eclipse.jdt.internal.compiler.Compiler.handleInternalException(Compiler.java:678)
	at org.eclipse.jdt.internal.compiler.Compiler.compile(Compiler.java:522)
	at play.classloading.ApplicationCompiler.compile(ApplicationCompiler.java:292)
	at play.classloading.ApplicationClasses$ApplicationClass.compile(ApplicationClasses.java:289)
	at play.classloading.ApplicationClassloader.detectChanges(ApplicationClassloader.java:326)
	at play.Play.detectChanges(Play.java:636)
	at play.Invoker$Invocation.init(Invoker.java:199)
	at play.jobs.Job.call(Job.java:185)
	at Invocation.Job(Play!)
[2016-09-21 16:16:51,141] ERROR [jobs-thread-9] play.Logger.niceThrowable(Logger.java:604) - 

@71ep5n6fc
Error during job execution (jobs.TaskKeeper2)

Compilation error (In /app/models/Task3.java around line 647)
The file /app/models/Task3.java could not be compiled. Error raised is : The operator += is undefined for the argument type(s) Long, double

play.exceptions.CompilationException: The operator += is undefined for the argument type(s) Long, double
	at play.classloading.ApplicationCompiler$2.acceptResult(ApplicationCompiler.java:256)
	at org.eclipse.jdt.internal.compiler.Compiler.handleInternalException(Compiler.java:678)
	at org.eclipse.jdt.internal.compiler.Compiler.compile(Compiler.java:522)
	at play.classloading.ApplicationCompiler.compile(ApplicationCompiler.java:292)
	at play.classloading.ApplicationClasses$ApplicationClass.compile(ApplicationClasses.java:289)
	at play.classloading.ApplicationClassloader.detectChanges(ApplicationClassloader.java:326)
	at play.Play.detectChanges(Play.java:636)
	at play.Invoker$Invocation.init(Invoker.java:199)
	at play.jobs.Job.call(Job.java:185)
	at Invocation.Job(Play!)
[2016-09-21 16:18:00,814] ERROR [main] play.Logger.error(Logger.java:426) - Could not bind on port 80
[2016-09-21 16:27:43,576] ERROR [play-thread-1] com.aton.base.BaseController.handleServerError(BaseController.java:232) - =======>Server Fatal Error<======
[2016-09-21 16:27:43,576] ERROR [play-thread-1] com.aton.base.BaseController.handleServerError(BaseController.java:233) - The validated expression is false
java.lang.IllegalArgumentException: The validated expression is false
	at org.apache.commons.lang.Validate.isTrue(Validate.java:158)
	at models.Task3.pay(Task3.java:703)
	at controllers.TaskPublish.confirmPayment3(TaskPublish.java:849)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:39)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:25)
	at java.lang.reflect.Method.invoke(Method.java:597)
	at play.mvc.ActionInvoker.invokeWithContinuation(ActionInvoker.java:524)
	at play.mvc.ActionInvoker.invoke(ActionInvoker.java:475)
	at play.mvc.ActionInvoker.invokeControllerMethod(ActionInvoker.java:451)
	at play.mvc.ActionInvoker.invokeControllerMethod(ActionInvoker.java:446)
	at play.mvc.ActionInvoker.invoke(ActionInvoker.java:160)
	at play.server.PlayHandler$NettyInvocation.execute(PlayHandler.java:267)
	at play.Invoker$Invocation$1.apply(Invoker.java:290)
	at play.Invoker$Invocation$1.apply(Invoker.java:287)
	at play.db.jpa.JPA.withTransaction(JPA.java:258)
	at play.db.jpa.JPA.withinFilter(JPA.java:217)
	at play.db.jpa.JPAPlugin$TransactionalFilter.withinFilter(JPAPlugin.java:299)
	at play.Invoker$Invocation.withinFilter(Invoker.java:271)
	at play.Invoker$Invocation.run(Invoker.java:287)
	at play.server.PlayHandler$NettyInvocation.run(PlayHandler.java:245)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:439)
	at java.util.concurrent.FutureTask$Sync.innerRun(FutureTask.java:303)
	at java.util.concurrent.FutureTask.run(FutureTask.java:138)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$301(ScheduledThreadPoolExecutor.java:98)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:206)
	at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:895)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:918)
	at java.lang.Thread.run(Thread.java:662)
[2016-09-21 16:27:50,886] ERROR [play-thread-1] com.aton.base.BaseController.handleServerError(BaseController.java:232) - =======>Server Fatal Error<======
[2016-09-21 16:27:50,886] ERROR [play-thread-1] com.aton.base.BaseController.handleServerError(BaseController.java:233) - The validated expression is false
java.lang.IllegalArgumentException: The validated expression is false
	at org.apache.commons.lang.Validate.isTrue(Validate.java:158)
	at models.Task3.pay(Task3.java:703)
	at controllers.TaskPublish.confirmPayment3(TaskPublish.java:849)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:39)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:25)
	at java.lang.reflect.Method.invoke(Method.java:597)
	at play.mvc.ActionInvoker.invokeWithContinuation(ActionInvoker.java:524)
	at play.mvc.ActionInvoker.invoke(ActionInvoker.java:475)
	at play.mvc.ActionInvoker.invokeControllerMethod(ActionInvoker.java:451)
	at play.mvc.ActionInvoker.invokeControllerMethod(ActionInvoker.java:446)
	at play.mvc.ActionInvoker.invoke(ActionInvoker.java:160)
	at play.server.PlayHandler$NettyInvocation.execute(PlayHandler.java:267)
	at play.Invoker$Invocation$1.apply(Invoker.java:290)
	at play.Invoker$Invocation$1.apply(Invoker.java:287)
	at play.db.jpa.JPA.withTransaction(JPA.java:258)
	at play.db.jpa.JPA.withinFilter(JPA.java:217)
	at play.db.jpa.JPAPlugin$TransactionalFilter.withinFilter(JPAPlugin.java:299)
	at play.Invoker$Invocation.withinFilter(Invoker.java:271)
	at play.Invoker$Invocation.run(Invoker.java:287)
	at play.server.PlayHandler$NettyInvocation.run(PlayHandler.java:245)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:439)
	at java.util.concurrent.FutureTask$Sync.innerRun(FutureTask.java:303)
	at java.util.concurrent.FutureTask.run(FutureTask.java:138)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$301(ScheduledThreadPoolExecutor.java:98)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:206)
	at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:895)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:918)
	at java.lang.Thread.run(Thread.java:662)
